alert-common/src/main/java/com/synopsys/integration/alert/common/enumeration/FormatType.java

```diff
@@ -23,8 +23,23 @@
 package com.synopsys.integration.alert.common.enumeration;
 
 public enum FormatType {
-    DEFAULT,
-    DIGEST,
-    SUMMARY
+    DEFAULT("Default", "The message will contain all the relevant data found in your selected provider."),
+    DIGEST("Digest", "The message will contain a delta of the content found in your selected provider since it was last queried."),
```

Comment:
Can we change the Digest format to "The message will contain a delta of the content found in your selected provider since it was last queried. Add and Delete operations will cancel each other out depending on the order they occurred."